sass/components/_image_compare.scss

```diff
@@ -1,3 +1,11 @@
+p.image-compare-instruction {
+  font-weight: bold;
+  margin-bottom: 0;
+}
+div.image-compare-container {
+  max-height: 720px;
+  aspect-ratio: 16 / 9;
```

Comment:
**Nit**: this probably can be removed? The shortcode adds the `aspect-ratio` via style attribute.

```suggestion
```